labelbox/client.py

```diff
@@ -1786,3 +1789,104 @@ def get_batch(self, project_id: str, batch_id: str) -> Entity.Batch:
             experimental=True)["project"]["batches"]["nodes"][0]
 
         return Entity.Batch(self, project_id, batch)
+
+    def send_to_annotate_from_catalog(
+            self,
+            batch_name: str,
+            data_rows: DataRowIdentifiers,
+            source_model_run_id: Optional[str],
+            source_project_id: Optional[str],
+            destination_project: str,
+            task_queue_id: Optional[str],
+            exclude_data_rows_in_project: bool = False,
+            ontology_mapping: Dict[str, str] = None,
+            override_existing_annotations_rule:
+        ConflictResolutionStrategy = ConflictResolutionStrategy.KeepExisting,
+            priority: int = 5):
```

Comment:
Let's add docstrings to both send_to_annotate methods with descriptions for what all of the parameters do